Makefile

```diff
@@ -238,10 +238,9 @@ update_submodules:
 ############################################################
 
 lock_deps: lock_pip
-	conda-lock -f environment.yml
-	conda-lock render
```

Comment:
The render argument and the output it produced (the multi-platform conda-lock.yml file) was not actually used in making our environments.